visma/matrix/operations.py

```diff
@@ -220,3 +220,26 @@ def scalarDiv(const, mat):
         return matRes
     else:
         logger.error("ZeroDivisionError: Cannot divide matrix by zero")
+
+
+def traceMat(mat):
+    """Returns the trace of a square matrix (sum of diagonal elements)
+
+    Arguments:
+        mat {visma.matrix.structure.Matrix} -- matrix token
+
+    Returns:
+        trace {visma.matrix.structure.Matrix} -- 1x1 Matrix token
+    """
+    trace = Matrix()
```

Comment:
You can return a tokens list instead of 1x1 matrix for trace. Also define the trace() inside the **_SquareMat_** class.
```suggestion
    trace = []
```